vortex-gpu/src/bit_unpack.rs

```diff
@@ -33,24 +31,18 @@ impl UnpackKernelId {
     }
 }
 
-static CUDA_KERNELS: LazyLock<RwLock<HashMap<UnpackKernelId, CudaFunction>>> =
-    LazyLock::new(|| RwLock::new(HashMap::new()));
-
 fn cuda_bit_unpack_kernel(
     kernel_id: UnpackKernelId,
     ctx: Arc<CudaContext>,
 ) -> VortexResult<CudaFunction> {
-    if let Some(kernel) = CUDA_KERNELS.read().get(&kernel_id) {
```

Comment:
I assume we cached compiled kernels before. Just for my understanding, why do we drop this? Can kernels with the same id ever be different?